app/gui/src/model/execution_context.rs

```diff
@@ -3,6 +3,7 @@
 use crate::prelude::*;
 
 use crate::model::module::QualifiedName as ModuleQualifiedName;
+use crate::model::suggestion_database::entry::QualifiedName as SuggestionQualifiedName;
```

Comment:
use qualified names instead. Here you can use `suggestion::QualifiedName` instead.